mscore/musescore.cpp

```diff
@@ -496,11 +507,16 @@ bool MuseScore::importExtension(QString path)
             return false;
             }
       zipFile3.close();
+      msgBox->hide();
 
       mscore->reloadInstrumentTemplates();
       mscore->updateNewWizard();
       mscore->updateInstrumentDialog();
 
+      msgBox->setWindowTitle(tr("Packages installer"));
+      msgBox->setText(tr("<p align='center'>Please wait, loading soundfonts... </p>"));
```

Comment:
Can we separate the HTML from the text to translate here? Like
`msgBox->setText("<p align='center'>%1</p>").arg(tr("Please wait, loading soundfonts..."));``

Reply:
ah, yes, thank you!